sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -1511,7 +1514,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
           {
             if (path.size() == 1 && path[0] == XmlTagName::c_Enabled)
             {
-              result.Enabled = node.Value;
+              result.Enabled = (node.Value == std::string("true"));
```

Comment:
You're using `std::strcmp`  in the code below, be consistent.

Reply:
Nice catch. Resolved in the next commit.